### PR TITLE
Modify/edit the existing content

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.8+

Issue: None

Link to docs preview: https://53165--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/index.html

QE review: Not required
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: post_installation_configuration/index.adoc [hunk 1]
```diff
@@ -63,23 +63,27 @@ As a cluster administrator, you can perform the following operations with the ma
...
 ** *Dynamic provisioning*: You can dynamically provision storage on demand by defining and creating storage classes that control different levels of storage, including storage access.
 
-** *Static provisioning*: Cluster administrators can use Kubernetes persistent volumes to make existing storage available to a cluster by supporting various device configurations and mount options.
+** *Static provisioning*: You can use Kubernetes persistent volumes to make existing storage available to a cluster. Static provisioning can support various device configurations and mount options.
 
 * xref:../post_installation_configuration/preparing-for-users.adoc#post-install-preparing-for-users[Configure users]:
-OAuth access tokens allow users to authenticate themselves to the API. As a cluster administrator, you can configure OAuth to specify an identity provider, use role-based access control to define and apply permissions to users, and install an Operator from OperatorHub.
+OAuth access tokens allow users to authenticate themselves to the API. As a cluster administrator, you can configure OAuth to perform the following tasks:
+
+* Specify an identity provider
```
Comment (Amrita42) [inline post_installation_configuration/index.adoc:84]: Please add + so it renders as part of the sentence

Comment (Amrita42) [reply post_installation_configuration/index.adoc:84]: otherwise looks good , nice content improvement

Comment (xenolinux) [reply post_installation_configuration/index.adoc:84]: acked

**Pull request merged**